src/test/java/unrc/dose/UserStatTest.java

```diff
@@ -24,21 +24,76 @@ public static void beforeAll() {
 		log.info("UserStatTest BeforeClass");
 		Base.open();
 	}
-	
-	@Before
-	public void beforeTest() {
-		Base.openTransaction();
-	}
 
 	@AfterClass
 	public static void afterAll() {
 		log.info("UserStatTest AfterClass");
 		Base.close();
 	}
-	
-	@After
-	public void afterTest() {
-		Base.rollbackTransaction();
+
+	/**
+	 * Test the method getUserId from
+	 * UserStat class.
+	 */
+	@Test
+	public void getUserIdTest() {
+		User u = new User();
```

Comment:
Use the method 

public static User set(final String name, final String pass,
        final String email, final Boolean admin)

from the User class to create a new User.